src/plugins/event-plugins/__integ__/ResourcePlugin.test.ts

```diff
@@ -18,25 +18,25 @@ test('when resource loads before the plugin then the resource is recorded', asyn
     const events = JSON.parse(await REQUEST_BODY.textContent).RumEvents.filter(
         (e) =>
             e.type === PERFORMANCE_RESOURCE_EVENT_TYPE &&
-            JSON.parse(e.details).targetUrl.includes(
-                'rum_javascript_telemetry.js'
-            )
+            JSON.parse(e.details).targetUrl.includes('blank.png')
     );
 
     await t.expect(events.length).eql(1);
 });
 
-test('when resource loads after window.load then the resource is recorded', async (t: TestController) => {
+test('when resource loads before the plugin then the resource is recorded', async (t: TestController) => {
```

Comment:
Question: When does window.load occur during TestCafe test? Why did reversing these fix the problem?

Reply:
Firefox seems caching the image between runs, so during the second run it doesn't make a request to the server for it.

I assume there is some Firefox (or TestCafe) configuration that prevents this caching.

Reply:
Switched the tests in #448 like you did here